docs/releasenotes/5.2.0.rst

```diff
@@ -0,0 +1,28 @@
+5.2.0
+-----
+
+API Changes
+===========
+
+Deprecations
+^^^^^^^^^^^^
+
+These version constants have been deprecated. ``VERSION`` will be removed in
+Pillow 6.0.0, and ``PILLOW_VERSION`` will be removed in a future release.
```

Comment:
Doesn't it mean that PILLOW_VERSION will be removed in a future release i.e. in 5.3, before 6.0? For me, "and ``PILLOW_VERSION`` will be removed after that." looks more clear.

Reply:
It is ambiguous. Updated!